prombench/README.md

```diff
@@ -94,6 +94,7 @@ cat $AUTH_FILE | base64 -w 0
 ## Usage
 
 ### Trigger tests via a Github comment.
+<!-- If you change the heading, please change the anchor at 7a_commentmonitor_configmap_noparse.yaml aswell. -->
```

Comment:
```suggestion
<!-- If you change the heading, also change the anchor in the comment monitor config map. -->
```

no need to be that specific. Error prone as filenames change quite often and there is no way you would remember to update the filename here as well.